Makefile.core.mk

```diff
@@ -143,10 +143,18 @@ export
 ##@ Testing
 
 .PHONY: test
-test: envtest ## Run unit tests.
+test: envtest test.unit test.integration ## Run both unit tests and integration test.
```

Comment:
Since envtest is already a dependency of `test.integration`, we don't need to list it as a dependency here.
```suggestion
test: test.unit test.integration ## Run both unit tests and integration test.
```